web-app/HeimScripts/heatmap/run.r

```diff
@@ -7,6 +7,8 @@ main <- function(max_rows=50){
   if(ncol(df) > 3){
     variances <- apply(df[,3:ncol(df)],1,var) # Calculating variance per probe
     df["SIGNIFICANCE"] <- variances
+    df["MEAN"] <- apply(df[,3:ncol(df)],1,mean, na.rm = T)
```

Comment:
FYI there is also a function for this rowMeans. https://stat.ethz.ch/R-manual/R-devel/library/base/html/colSums.html : "These functions are equivalent to use of apply with FUN = mean or FUN = sum with appropriate margins, but are a lot faster. " . Probably the contribution to total running time of the script by this particular operation is minimal, but just FYI